pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "termgraph"
+version = "0.1.0"
+description = "Terminal candle stick graph library"
+authors = ["Sebastian Geisler <github@gnet.me>"]
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
Anaconda Python currently provide Python 3.8.
Maybe having Python minimal version to 3.8 may be enough

Reply:
Good catch, I'm only testing 3.5-3.7, so I'll make this 3.5 and add 3.8 and 3.9 to the test matrix.